marda_registry/data/extractors/neware-nda-reader.yaml

```diff
@@ -0,0 +1,50 @@
+---
+id: >-
+    NewareNDA 
+name: >-
+    NewareNDA
+description: >-
+    Python module and command line tool for reading and converting Neware nda and ndax battery cycling data files. 
+supported_filetypes:
+    - id: neware-nda
+license:
+    spdx: BSD-3-Clause
+    uri: https://github.com/Solid-Energy-Systems/NewareNDA/blob/master/LICENSE
+subject:
+    - electrochemistry
+citations:
+    - uri: https://github.com/Solid-Energy-Systems/NewareNDA
+      creators:
+          - D. Cogswell
+      title: NewareNDA github repository
+      type: software
+source_repository: https://github.com/Solid-Energy-Systems/NewareNDA
+usage:
+    - method: cli
+      command: NewareNDA-cli.py {{ input_path }} --format csv {{ output_path }}
```

Comment:
Hm, what is the python object generated from `NewareNDA.read()` below? Is it a dict, or some other object? I think we should probably have the analogous CLI usage (json?) for the python usage for now, instead of having 10 equivalent ones with an output format.

Also, we have `output_type` in the template: https://marda-alliance.github.io/metadata_extractors_schema/main/mme_schema/UsageTemplate.html

Reply:
It returns a dataframe at the moment, the CLI then just calls the various `df.to_<x>()` functions that pandas has.

`output_type` requires a MaRDA filetype itself right? I think it's also single-valued (at the moment) so not sure how useful it is here

Reply:
OK, then we probably need to mention pandas somewhere. I'll think about how to add that into schema (also for the xarray requirement in yadg) in another PR.

And yes, we should probably add those as `FileTypes` into the registry anyway.